### PR TITLE
test(sbb-map-container): add visual spec

### PR DESCRIPTION
**Pull request by TomMenga**

### CODE REVIEW

File: src/elements/map-container/map-container.visual.spec.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+import '../title.js';
+
+describe(`sbb-map-container`, () => {
+  describeViewports({ viewports: ['zero', 'medium'], viewportHeight: 500 }, () => {
+    it(
+      visualDiffDefault.name,
+      visualDiffDefault.with(async (setup) => {
+        await setup.withFixture(
+          html`
+            <sbb-header expanded hide-on-scroll>
```
Comment (jeripeierSBB) [inline src/elements/map-container/map-container.visual.spec.ts:17]: Does it make a difference if hide-on-scroll is set or not?

**Review by jeripeierSBB — APPROVED** [review]
LGTM (not checked the outcome)

File: src/elements/map-container/map-container.visual.spec.ts (hunk 1 above)
Comment (TomMenga) [reply src/elements/map-container/map-container.visual.spec.ts:17]: mmm no, it's a leftover from the tentative of testing the scroll behavior :/

Comment (jeripeierSBB) [reply src/elements/map-container/map-container.visual.spec.ts]: maybe you could remove it to not generate wrong assumptions, but not important

**Pull request merged**